exporter/azuremonitorexporter/README.md

```diff
@@ -26,7 +26,12 @@ Example:
 ```yaml
 exporters:
   azuremonitor:
-    instrumentation_key: b1cd0778-85fc-4677-a3fa-79d3c23e0efd
+    instrumentation_key: ########-####-####-####-############
+
+service:
+  pipelines:
+    traces:
+      exporters: [azuremonitor, <other-exporters>]
```

Comment:
Maybe add a receiver too? Otherwise this pipeline is invalid.

Reply:
I'm not sure I understand correctly. In the [config file](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-demo/values.yaml), `otlp` is already added to `receivers`. I only had to add `azuremonitor` to `exporters` to make it work.

Reply:
I think most component readme files only include the configuration of the relevant component.  It is expected that the user knows they need to include the component in a pipeline in order for it to be used and avoids including large amounts of boilerplate that may confuse users who are using different components for other parts of their pipeline.

Reply:
The official [Kubernetes deployment](https://opentelemetry.io/docs/demo/kubernetes-deployment/#bring-your-own-backend) documentation shows exporters being added to both `exporters` and `service`, which was helpful to me. If you think this detail is unnecessary and might just be confusing to users, I can close this PR.

<img width="394" alt="config-exporter-example" src="https://user-images.githubusercontent.com/27689846/221981136-3079188f-2b66-4c3e-9fde-b5934eb81dd4.png">